src/DotNet.Status.Web/appsettings.Production.json

```diff
@@ -14,6 +14,7 @@
     "Repository": "core-eng",
     "NotificationTarget": "dotnet/dnceng",
     "AlertLabels": [ "First Responder", "Critical" ],
+    "EnvironmentLabels": [],
```

Comment:
Won't this mean prod still finds staging, since it won't know not to search for it?

Reply:
Yeah, you're totally right. Can we add a "Public" label or we don't want that?

Reply:
I'd call it "Production", since that's what most of the other pieces of code call the two environments. Staging/Production.

Reply:
Remember to actually create the label in core-eng, otherwise it won't be able to get applied by this code.  I'm not sure what color to make it... probably the same gray color as the Staging one (since "Active Alert" is already red, we don't need more scary colors)

Reply:
@ilyas1974 expreseed the wish to have "Environment" in the name as well. I think we might be fine with keeping it Staging and adding Production.

What do you think?

Reply:
Ok, I named it `Production`, since we already have `Staging` and I added description so it is clear